### DISCUSSION TITLE
Tampermonkey Safari Promocodes

### DISCUSSION THREAD

**Discussion by derjanb** [Tampermonkey/tampermonkey#2251] [Show and tell]
# Safari Promocodes

Github users can here apply for a promocode for the Safari extension by posting the ID you got via mail after registering at https://tmnk.net/promo.php

The only requirement is that your Github account contributed to Tampermonkey development by creating or commenting on an issue before today or is involved in userscript development.

I'll send you a promocode via mail as soon as possible, but be aware that only a small number of promocodes can be generated per day.

If you want to support the development of Tampermonkey you can also buy the extension at the [Mac App Store](https://apps.apple.com/us/app/tampermonkey/id6738342400) ;-).

**Comment by maxzhang666** [comment]
81e7a544-0040-4d22-8078-db88adc778dc

**Reply by wjyjw** [reply]
6f630ea0-8473-4e4e-8c14-ba8bf08bc9d2

**Comment by ming5chiu** [comment]
410ecd57-d210-4650-ab96-35a83d6f35b1

**Comment by Morikernel** [comment]
29d15fb2-2f65-4602-84c0-c399fb133e51

**Comment by 3118Badylai** [comment]
0773ac21-a812-457b-bd2a-287b1f765499

**Comment by Morikernel** [comment]
提示无效兑换码

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Tampermonkey/tampermonkey"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月1日(星期天) 上午10:16
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Tampermonkey/tampermonkey] Tampermonkey Safari Promocodes (Discussion #2251)





 
0773ac21-a812-457b-bd2a-287b1f765499

**Comment by derjanb** [comment]
@Sunshine214000 

You never got one, because I don't know what script you're developing.

**Comment by Parsa-X** [comment]
ID 6b769a3f-c0fa-477b-a300-60a5968e8dd1

**Comment by Affhamra** [comment]
6a434900-3f42-40c1-a8d4-01d7fc99af74

**Comment by githubdulong** [comment]
a27947c0-2e1a-49dd-b203-89eaa9b88f52

**Comment by marcopaselli** [comment]
e7d592d4-57e9-4a9d-bc3b-4caccc46436d

**Comment by aswinrajbk** [comment]
8a439f4c-0f18-4ff1-88f1-135708b98cc7

**Comment by ghost** [comment]
600ae3a0-2297-4246-8018-c5fba5c0d37

**Comment by sergeyn777** [comment]
f7cc5ba5-8873-4acf-83e3-7244438d6dfb

**Comment by sergeyn777** [comment]
f7cc5ba5-8873-4acf-83e3-7244438d6dfb

**Reply by aswinrajbk** [reply]
Tbh what is this?

**Reply by sergeyn777** [reply]
Not sure I was trying to to get a code for tempermonkey. It's said type ur id and send in github

**Comment by nasonnsir** [comment]
5dee4f34-9d3a-4aa5-9dc5-044e95ca66d8

**Comment by torin-stephen** [comment]
28a97d89-621e-44c6-8019-f5af404b6a33

**Comment by xmdltz** [comment]
53a362af-7ee8-4dc5-843e-1b93d8ffc8f0

**Reply by h4rl7m** [reply]
did you ever get it

**Comment by h4rl7m** [comment]
9137b658-886a-4ffb-9d0e-189b84111b58

**Comment by gesiel-alt** [comment]
76296b76-83bb-4371-8499-1ead18d03acf

**Comment by 11liju** [comment]
4633a0dc-56da-4341-8a9f-9d64b1f5e0a5

**Comment by k19890703** [comment]
273f0276-54f8-476e-bb5b-2b15f04c01e8

**Comment by derjanb** [comment]
Free promo codes are intended for Tampermonkey supporters and userscript developers. You can prove that, for example by owning a userscript repository. @k19890703 you don't have a single repository and even registered at GitHub just recently. 

BTW: just forking a userscript repository is not necessarily development. If needed, show me a merge request where your contributed to a userscript. 

Thanks.

**Comment by abandon1013** [comment]
ID 26478998-520c-4bad-9ebc-e8c982b47dfd

**Comment by Snake221982** [comment]
ID 19d5d4ed-6da7-439c-aa0b-eaffe2d5e97b

**Comment by antongoated557** [comment]
ID 33694b60-70b8-433f-b632-a06602bdbedd

**Comment by derjanb** [comment]
This isn't meant as a solicitation for new Github accounts. Where are the repositories for the userscripts you're developing?

**Reply by antongoated557** [reply]
Wdym I'm confused

**Comment by mlyxshi** [comment]
125cf86b-83c9-4a65-910e-2f2570a40aa3

**Comment by xNeros** [comment]
ID 8f73996c-e5bd-4bdf-9142-33736f730742

**Comment by AndyCort** [comment]
ae662ea9-d601-4019-ba5a-c61908e34cef

**Comment by InsomniaCoding** [comment]
6e5a7710-6109-4918-b142-f844fc8d3e88

**Comment by KiligWYu** [comment]
5cf540b6-382b-4399-9db7-c51a8c95ed72

**Comment by AeongKim** [comment]
708c065c-1882-4e88-99fa-a68721c3dad8

**Reply by qyld1219** [reply]
你收到了吗？

**Comment by bol13mn** [comment]
bdc4f0d6-80b4-4ede-8409- aa6ec49004e4

**Comment by wjyjw** [comment]
6f630ea0-8473-4e4e-8c14-ba8bf08bc9d2

**Comment by cliff-cc** [comment]
badd4030-9865-448d-a1b9-a56b0ffae461

**Comment by ZacharyHu0** [comment]
3e526ac9-cd21-4812-ba64-c29142594b16

**Comment by mimethegap** [comment]
ce19253c-2082-44f8-9284-cf58bd5e3739

**Comment by Haku-taku** [comment]
912adf19-0ed7-441a-939d-5cab0f3c3bbd

**Comment by boitar** [comment]
1c7fc2af-ca94-434b-8289-b0fc9b858dff

**Comment by xp9477** [comment]
d93db0a7-ce30-45d9-a0c4-a1438bccc2bd

**Comment by jb9479385** [comment]
ID 056c0087-4808-44dd-9d70-a5b5981aa208

**Comment by checion** [comment]
dac3a652-7229-4212-848d-d3bbaeaf1138

**Comment by landrarwolf** [comment]
a6620350-330e-4e1e-a777-d59fb02010fc

**Comment by Jerryhu028** [comment]
29898a70-9638-470c-b407-40f84bc2d0ba

**Comment by Spoou** [comment]
c9796960-24c1-4293-aa18-89420757bb24

**Comment by Mr-Shadow-Ru** [comment]
12501475-19f9-42eb-9968-be461f1d4cd2

**Comment by sierralx** [comment]
808f21eb-235c-4436-878b-21d2758eb721

**Comment by vash-fiery** [comment]
f65eaa7c-8a0c-410e-b1e7-d16995d1e955

**Comment by cfvgbhn123** [comment]
58cbb4e3-5ade-46a1-b908-99d75cef1ebc

**Comment by JeffersonHora** [comment]
0faaf04c-379d-4a40-9aad-7c17f87eb0c5

**Comment by spillins1** [comment]
92efeff-41fe-4b05-a1cb-8550970f4386

**Comment by Aboudi1100** [comment]
e4aeb6ae-be70-4925-8ad1-ebfa743f9b29

**Comment by deevroman** [comment]
465aed2b-f2d0-4c8e-9a3c-56ec90b478c2

**Comment by danny2027** [comment]
a45cdfe0-edc8-4326-8577-8cbff7bf1cdd

**Comment by saaam001** [comment]
6638e34f-2342-4b29-8d1f-b5c768ec5a47

**Reply by Monica970291** [reply]
d3680780-d70d-4973-8644-3b3661d25280

**Comment by k19890703** [comment]
在 2025年9月3日 星期三 上午04:48:46 [GMT+8]， ***@***.***> 寫道：  
 
 


d3680780-d70d-4973-8644-3b3661d25280

**Comment by Monica970291** [comment]
d3680780-d70d-4973-8644-3b3661d25280

ID:amwayant1011@gmail.com

**Comment by Kiriko-o** [comment]
6155b1bb-7a07-4b63-af95-bd572bd94e0b

**Comment by BubblingXuYijie** [comment]
06e7aed2-cfbd-4e3a-8b57-c92fa43782fc

**Comment by Kiriko-o** [comment]
？你发给我啥意思 徐桑

2025年10月2日(木) 14:57 徐一杰 ***@***.***>:

**Reply by BubblingXuYijie** [reply]
...宝贝，这下面任何一个人发评论你都会收到通知

**Comment by Kiriko-o** [comment]
好的谢谢宝贝 我以为我被爆破了🥴

2025年10月2日(木) 15:02 徐一杰 ***@***.***>:

**Comment by LesleyPhilippaerts-arch** [comment]
3cad2a79-d956-462d-9807-8546195cecea

**Comment by mila3224222-arch** [comment]
206cba6e-2e51-47d5-815b-c20c936e0f4b

**Comment by LesleyPhilippaerts-arch** [comment]
Hey

Where can i redeem the code? 
Lesley Philippaerts

> Op 18 okt 2025 om 13:23 heeft mila3224222-arch ***@***.***> het volgende geschreven:
> 
> 206cba6e-2e51-47d5-815b-c20c936e0f4b

**Comment by kalerpark** [comment]
d3c52712-26bc-4a54-b48b-173c16b8c1b3

**Comment by hussamrashid-droid** [comment]
ID f27ad08d-b611-4425-b936-9d7305feb2fb

**Comment by DeviateFromThePlan** [comment]
9e804885-b959-4c51-83aa-48cd094b6e25

**Comment by wei83109-coder** [comment]
fd0972c0-ad49-48c1-98d9-40c8fffccab3

**Comment by ChingningLam** [comment]
6a9c66c7-a259-4a92-8214-4aa46adfd9f5

**Reply by qyld1219** [reply]
你收到了吗？

**Comment by qyld1219** [comment]
2972​​5361-ef7e-415b-a663-fc40b9493980

**Reply by qyld1219** [reply]
qyld1219@gmail.com

**Comment by qyld1219** [comment]
会发送到哪里？

2972​​5361-ef7e-415b-a663-fc40b9493980

**Reply by derjanb** [reply]
Please check this post for the requirements to get a code: https://github.com/Tampermonkey/tampermonkey/discussions/2251#discussion-7512538

**Comment by maamirnaeem** [comment]
a424786b-b2da-4b73-bf86-1174634f48d2

**Comment by hosseintajdari** [comment]
03b44067-e06d-4079-9b55-74f10ae945a3

Testing and analysis

**Comment by xNeros** [comment]
8f73996c-e5bd-4bdf-9142-33736f730742

**Comment by heinzgitz** [comment]
f445a2ce-77e4-4e44-beb7-165685c7a4bf

**Comment by denny83** [comment]
a810e185-d539-4550-afb5-d36846705cfc

**Comment by ygncjx** [comment]
f89e9b56-fd0c-45df-b006-b9beba3ab65a

**Comment by jfrostbite** [comment]
390398bd-0f95-44b9-b0f5-48e42e078086

**Comment by JellyFangThunderBucket** [comment]
26c43e77-7a0e-4eb6-a1f8-ff571756eeb6

**Comment by JDavydovPortfolio** [comment]
4ccd3ad1-af7a-4bae-8207-bac1c82bb09d

**Comment by rtazyukov-png** [comment]
05d69a0c-083e-4f50-9243-20659f214966

**Comment by hezhen020** [comment]
369ca9f7-df64-4520-91ec-6056c41c0d36

**Comment by DJGCMUSIC** [comment]
7403b46e-231f-4954-8666-2ddcefa663fb

**Comment by 16ds-code** [comment]
76a7b22c-5f54-4dc5-a8ee-a430f73de60f

**Comment by f9sp9fhzyj-arch** [comment]
75fb23d3-4738-473e-b25a-14e148f6217e

**Comment by sakva520** [comment]
fe390e2e-a2d1-4825-9603-4348c7ab7a8c

**Comment by 261237407** [comment]
8e2bf8f9-9ad3-43c6-a04e-3666906736b3

**Reply by 261237407** [reply]
Where can we get the promo code? I have issued the ID.

**Comment by chunhuaqiuyue976** [comment]
ecb66c7f-4f16-4c44-a98a-b5276a33dec5

**Comment by wl71966672-maker** [comment]
83fed58e-dc8c-44bf-83f0-8a66ce4def12

**Comment by Zzzcy-tx** [comment]
3aea3cba-6a50-4124-a4a1-851f2695e455

**Comment by chunhuaqiuyue976** [comment]
3aea3cba-6a50-4124-a4a1-851f2695e455

**Comment by curtinp118** [comment]
06c19be1-4572-40ad-b72e-bed3968c5386

**Comment by chunhuaqiuyue976** [comment]
3aea3cba-6a50-4124-a4a1-851f2695e455

**Comment by rodek2006** [comment]
6369a805-8bb1-405a-93c3-9c75d5a3b207

**Comment by YankyAppel** [comment]
ID f5d4aec7-f487-4be7-8bda-6400838a0c5d

**Comment by StylesClashNation** [comment]
4b63707d-099c-418f-9a4e-90800cb47ada

**Comment by latorrews** [comment]
bbb18828-9a0a-4a7b-815e-1940210a30bf

**Comment by LeQuangHuy-1809** [comment]
e0bf82f4-f780-4f5a-86a3-5d6c4f9c0e9d

**Comment by Wen921006** [comment]
7d7b1621-84aa-4d87-85d1-21e58021988d
